examples/va/nfv/sriov/edpm/values.yaml

```diff
@@ -94,18 +34,15 @@ spec:
         service_net_map:
           nova_api_network: internal_api
           nova_libvirt_network: internal_api
-        # CHANGEME
         timesync_ntp_servers:
           - hostname: clock.redhat.com
         # CPU pinning settings
-        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47" # CHANGEME
+        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47"
         edpm_tuned_profile: "cpu-partitioning" 
```

Comment:
kind of a meta comment, with the focus on improving power saving
i am not convinced we should default to cpu-partioning in general
using "cpu-partitioning-powersave" would be a better default for all our VAs
"cpu-partitioning" disables all CPU power management on the host by forcing the performance governor which pins the CPU at 100% frequency

cpu-partitioning-powersave removes explicitly setting the governor and adds 
intel_pstate=passive to the the kernel command line allowing the cpus to idel when not in use.

https://github.com/redhat-performance/tuned/blob/master/profiles/cpu-partitioning-powersave/tuned.conf

it inhrits form cpu-partitioning so its otherwise effectively the same
it was introduced by the nfv dfg at the direct request of some our our larger telcos during the pandemic sue to the rising cost of power and there desire to reduce carbon output as well as cost.

we are planning to ship with https://specs.openstack.org/openstack/nova-specs/specs/2023.1/implemented/libvirt-cpu-state-mgmt.html enabled by default so nova will turn off unused cpu_dedicated_set cores out of the box to save power but ti does not hurt to also use the power saving tuned profile for the rest of the cores that nova will not manage.

Reply:
Ack.  Switching to `cpu-partitioning-powersave`.

Reply:
Fixed